packages/react-native-win32/overrides.json

```diff
@@ -326,6 +326,22 @@
       "baseHash": "a1d045fb2d0fb751d8f0518fd40af63454b88b04",
       "issue": "LEGACY_FIXME"
     },
+    {
+      "type": "derived",
```

Comment:
Could you mark these as patch? At some point I'm wanting to play with storing those as diffs so we can more easily tell what has changed.